.github/workflows/main.yml

```diff
@@ -13,9 +13,9 @@ jobs:
     with:
       linux_5_9_arguments_override: "-Xswiftc -warnings-as-errors --explicit-target-dependency-import-check error"
       linux_5_10_arguments_override: "-Xswiftc -warnings-as-errors --explicit-target-dependency-import-check error"
```

Comment:
We have to lose warnings-as-errors on 5.10, as we can no longer maintain it with this change.